### PR TITLE
[hawthorn][ironwood] Fix ora2 filesystem bug

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

ORA2 was not functional with the filesystem backend on `hawthorn` and `ironwood`.

## Proposal

- [x] Add missing ORA2 configuration to cms config
- [x] Upgrade ORA2 to latest 2.2 release that fixes the filesystem backend

### CODE REVIEW

File: releases/hawthorn/1/oee/config/lms/root_urls.py [hunk 1]
```diff
@@ -8,9 +8,11 @@
 from django.conf.urls import include, url
 
 from lms.urls import urlpatterns  # pylint: disable=import-error
+from openassessment.fileupload.urls import urlpatterns as oraurlpatterns
```
Comment (jmaupetit) [inline releases/hawthorn/1/oee/config/lms/root_urls.py:11]:
```suggestion
from openassessment.fileupload.urls import urlpatterns as ora_urlpatterns
```

File: releases/hawthorn/1/oee/config/lms/root_urls.py [hunk 2]
```diff
@@ -8,9 +8,11 @@
...
 
 
 urlpatterns += [
     # Fonzie API urls
     url(r"^api/", include("fonzie.urls", namespace="fonzie")),
 ]
+urlpatterns += oraurlpatterns
```
Comment (jmaupetit) [inline releases/hawthorn/1/oee/config/lms/root_urls.py:18]:
```suggestion
urlpatterns += ora_urlpatterns
```

File: releases/ironwood/2/oee/config/lms/root_urls.py [hunk 3]
```diff
@@ -8,9 +8,11 @@
 from django.conf.urls import include, url
 
 from lms.urls import urlpatterns  # pylint: disable=import-error
+from openassessment.fileupload.urls import urlpatterns as oraurlpatterns
```
Comment (jmaupetit) [inline releases/ironwood/2/oee/config/lms/root_urls.py:11]: ditto?

File: docker-compose.yml [hunk 4]
```diff
@@ -73,6 +73,8 @@ services:
       - ./${FLAVORED_EDX_RELEASE_PATH:-releases/master/0/bare}/data/export:/edx/var/edxapp/export
       - ./${FLAVORED_EDX_RELEASE_PATH:-releases/master/0/bare}/config:/config
       - ./docker/files/usr/local/bin/auth_init:/usr/local/bin/auth_init
+      # Add lines here to override source code in a Python module e.g.:
+      #- ./src/edx-ora2/openassessment:/usr/local/lib/python2.7/dist-packages/openassessment
```
Comment (sampaccoud) [inline docker-compose.yml:77]: @jmaupetit I'm proposing to add this doc lines to docker-compose as I always need to lookup the path :pray:

Comment (jmaupetit) [reply docker-compose.yml:77]: Good idea!

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**